browser.js

```diff
@@ -78,9 +77,8 @@ function pbkdf2Sync (password, salt, iterations, keylen, digest) {
       }
     }
 
-    var destPos = (i - 1) * hLen
-    var len = (i === l ? r : hLen)
-    T.copy(DK, destPos, 0, len)
```

Comment:
maybe if you specify a single iteration and a length that is longer or shorter then the hash algo ?